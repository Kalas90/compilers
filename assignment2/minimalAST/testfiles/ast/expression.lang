int main() {
	result = a * b + c * d > 2 + (1 / g);
}
