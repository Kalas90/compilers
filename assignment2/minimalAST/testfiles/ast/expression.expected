Program
  List
    FunctionDec
      IdDecl
      List
      CompoundStmt
        List
          AssignStmt
            IdUse
            Greater
              Add
                Mul
                  IdUse
                  IdUse
                Mul
                  IdUse
                  IdUse
              Add
                IntUse
                Div
                  IntUse
                  IdUse
